solidity/test/isms/RateLimitedIsm.t.sol

```diff
@@ -62,6 +64,12 @@ contract RateLimitedIsmTest is Test {
         rateLimitedIsm.verify(bytes(""), encodedMessage);
     }
 
+    function test_verifyOnlyRecipient() external {
+        bytes memory _message = MessageUtils.build(ORIGIN);
```

Comment:
IIUC, this relies on the fact that message recipient checking is before amount checking right? I guess the expectRevert will surface future issues, but i dont love that the message isn't a "valid" message in the first place

Reply:
the ordering doesnt matter, a reverts a revert